### PR TITLE
Release v0.2.2-alpha

### PR DESCRIPTION
**Pull request by MrAlias**
### Added

- The `net/http` client instrumentor. ([#91](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/91))
- Context propagation to the `net/http` server instrumentation. ([#92](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/92))
- Simplified example of an HTTP service in `go.opentelemtry.io/auto/examples/rolldice`. ([#195](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/195))

### Changed

- Upgrade OpenTelemetry semantic conventions to v1.18.0. ([#162](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/162))
- Remove the HTTP path from span names in `net/http`, `gin-gonic/gin`, and `gorilla/mux` instrumentations. ([#161](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/161))
- Update generated offsets. ([#186](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/186))

### CODE REVIEW

**Comment by damemi** [comment]
think you need to update fixtures:

```
# -- 💥 values are not equal 💥 --
# expected : 
# actual   : diff --git a/test/e2e/gorillamux/traces.json b/test/e2e/gorillamux/traces.json
# index 471d7a3..d20d10f 100644
# --- a/test/e2e/gorillamux/traces.json
# +++ b/test/e2e/gorillamux/traces.json
# @@ -12,7 +12,7 @@
#            {
#              "key": "telemetry.auto.version",
#              "value": {
# -              "stringValue": "v0.2.1-alpha"
# +              "stringValue": "v0.2.2-alpha"
#              }
#            },
#            {
# @@ -104,7 +104,7 @@
#                ],
#                "kind": 3,
#                "name": "/users/foo",
# -              "parentSpanId": "",
# +              "parentSpanId": "xxxxx",
#                "spanId": "xxxxx",
#                "status": {},
#                "traceId": "xxxxx"
# --
```

the change in `parentSpanId` is weird though. not sure why that is

**Comment by damemi** [comment]
ok, I think there is some flake/race in the e2e...

```
# --- a/test/e2e/gorillamux/traces.json
# +++ b/test/e2e/gorillamux/traces.json
# @@ -104,7 +104,7 @@
#                ],
#                "kind": 3,
#                "name": "/users/foo",
# -              "parentSpanId": "xxxxx",
# +              "parentSpanId": "",
#                "spanId": "xxxxx",
#                "status": {},
#                "traceId": "xxxxx"
# --
```

**Comment by MrAlias** [comment]
> ok, I think there is some flake/race in the e2e...
> 
> ```
> # --- a/test/e2e/gorillamux/traces.json
> # +++ b/test/e2e/gorillamux/traces.json
> # @@ -104,7 +104,7 @@
> #                ],
> #                "kind": 3,
> #                "name": "/users/foo",
> # -              "parentSpanId": "xxxxx",
> # +              "parentSpanId": "",
> #                "spanId": "xxxxx",
> #                "status": {},
> #                "traceId": "xxxxx"
> # --
> ```

yeah ...

Something is acting weird.

**Comment by damemi** [comment]
@MikeGoldsmith looks like you added the parentSpan field in https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/91/commits/56cbc7d9e41df12cfadc7f418267fbf5e0cc8b18.. do you have any context about that change? Just trying to figure out where the race could be

**Comment by damemi** [comment]
I think the http server (receiving the request) isn't always picking up the http client (sending the request) as its parent span. Since they're in the same process, I wonder if https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/118 will fix it.

**Review by MikeGoldsmith — APPROVED** [review]
Pending an issue to be raised to investigate flaky http propagation in tests, looks good to me 👍🏻

**Comment by damemi** [comment]
Issue for the flake: https://github.com/open-telemetry/opentelemetry-go-instrumentation/issues/220

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,15 +8,19 @@ OpenTelemetry Go Automatic Instrumentation adheres to [Semantic Versioning](http
 
 ## [Unreleased]
 
+## [v0.2.2-alpha] - 2023-07-06
```
Comment (MrAlias) [inline CHANGELOG.md:11]:
```suggestion
## [v0.2.2-alpha] - 2023-07-11
```

**Pull request merged**